tools/devops/automation/run-post-ci-build-vs-insertion.yml

```
@@ -0,0 +1,86 @@
+# YAML pipeline for post build operations. 
+# This pipeline will sign the nugets and will prepare them for a release.
+
+trigger: none
+pr: none
+
+parameters:
+
+  - name: stageDisplayNamePrefix
+    type: string
+    default: ''
+
+  - name: repositoryAlias
+    type: string
+    default: self
+
+  - name: commit
+    type: string
+    default: HEAD
+
+  - name: pushNugets
+    type: boolean
+    default: true # default to true until otherwhise
+
+  - name: pushNugetsToMaestro
+    type: boolean
+    default: true
+
+# we cannot use a template in a pipeline context
+resources:
+  pipelines:
+    - pipeline: macios
+      source: \Xamarin\Mac-iOS\ci pipelines\xamarin-macios-ci
+      trigger:
+        branches:
+          include:
+            - main
+            - release-test/*
+            - release/*
+            - net7.0
+            - net8.0
+            - net9.0
+            - xcode16
```

Make it ready for next year:

```suggestion
            - xcode??
```